src/arguments/date.ts

```diff
@@ -0,0 +1,11 @@
+import { Argument, KlasaMessage, Possible } from 'klasa';
+
+export default class CoreArgument extends Argument {
+
+	public run(argument: string, possible: Possible, message: KlasaMessage): Date {
+		const date = new Date(argument);
+		if (!isNaN(date.getTime()) && date.getTime() > Date.now()) return date;
```

Comment:
why not be able to have dates *before* the current time?

Reply:
Not changing the existing behavior at this time. This piece can still be transferred and edited.